### PR TITLE
doc: edit agl-vms.rst adjust to v1.3

### PR DESCRIPTION
**Pull request by niluohe**

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

File: doc/tutorials/agl-vms.rst [hunk 1]
```diff
@@ -145,19 +145,24 @@ Service OS
...
-   OS bundles::
+#. Use `acrn_quick_setup.sh <https://github.com/projectacrn/acrn-hypervisor/blob/84c2b8819f479c5e6f4641490ff4bf6004f112d1/doc/getting-started/acrn_quick_setup.sh>`_ 
+   to automatic installation ACRN::
+      
+      # sh acrn_quick_setup.sh -s 31080 -i 
+
+#. After installation, the system will start up again automatically
+
+#. Reboot the system, choose "ACRN Hypervisor" and launch Clear Linux OS
+   SOS. If the EFI boot order is not right, use :kbd:`F10`
```
Comment (deb-intel) [inline doc/tutorials/agl-vms.rst]: Please remove this instance and all possible instances of SOS and UOS and replace with Service VM / User VM.

File: doc/tutorials/agl-vms.rst [hunk 2]
```diff
@@ -145,19 +145,24 @@ Service OS
...
+      # swupd verify --fix --picky -m 31080
 
-#. Use the ``swupd bundle-add`` command and add needed Clear Linux
-   OS bundles::
+#. Use `acrn_quick_setup.sh <https://github.com/projectacrn/acrn-hypervisor/blob/84c2b8819f479c5e6f4641490ff4bf6004f112d1/doc/getting-started/acrn_quick_setup.sh>`_ 
+   to automatic installation ACRN::
+      
+      # sh acrn_quick_setup.sh -s 31080 -i 
+
+#. After installation, the system will start up again automatically
```
Comment (deb-intel) [inline doc/tutorials/agl-vms.rst]:
```suggestion
#. After installation, the system will automatically start.
```

File: doc/tutorials/agl-vms.rst [hunk 3]
```diff
@@ -145,19 +145,24 @@ Service OS
...
    unintentionally update or change the Clear Linux OS version, you can
    fix it again using::
 
-      # swupd verify --fix --picky -m 26200
+      # swupd verify --fix --picky -m 31080
 
-#. Use the ``swupd bundle-add`` command and add needed Clear Linux
-   OS bundles::
+#. Use `acrn_quick_setup.sh <https://github.com/projectacrn/acrn-hypervisor/blob/84c2b8819f479c5e6f4641490ff4bf6004f112d1/doc/getting-started/acrn_quick_setup.sh>`_ 
+   to automatic installation ACRN::
```
Comment (deb-intel) [inline doc/tutorials/agl-vms.rst]:
```suggestion
   to automatically install ACRN::
```

File: doc/tutorials/agl-vms.rst [hunk 4]
```diff
@@ -243,25 +199,26 @@ In this demo, we use acrn-kernel as the baseline for development for AGL.
 
       $ cd workspace
       $ git clone https://github.com/projectacrn/acrn-kernel
-      $ git checkout tags/acrn-2018w49.3-140000p
-      $ make menuconfig
+      $ git checkout tags/acrn-2019w39.1-140000p
+      $ cp kernel_config_uos .config
+      $ make olddefconfig
+      $ vi .config
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]: Shouldn't this `.config` editing step be done *before* running `make olddefconfig`?

File: doc/tutorials/agl-vms.rst [hunk 5]
```diff
@@ -302,139 +259,139 @@ Setting up AGLs
...
+      #logger_setting, format: logger_name,level; like following
+      logger_setting="--logger_setting console,level=4;kmsg,level=3"
 
-      launch_agl 1 1 "64 448 8" 0x000F00 agl
+      #for memsize setting
+      mem_size=2048M
 
-#. Create the ``launch_rse.sh`` script for the AGL RSE VM with this
-   content::
+      acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge \
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]: Just double-checking: do we still have the `-c` option in that version of `acrn-dm`?

File: doc/tutorials/agl-vms.rst [hunk 6]
```diff
@@ -302,139 +259,139 @@ Setting up AGLs
...
-        #for memsize setting
-        mem_size=2048M
+      #logger_setting, format: logger_name,level; like following
+      logger_setting="--logger_setting console,level=4;kmsg,level=3"
 
+      #for memsize setting
+      mem_size=2048M
 
-        acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
+      acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge -U 495ae2e5-2603-4d64-af76-d4bc5a8ec0e5 \
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]: Same question re: `-c` parameter

File: doc/tutorials/agl-vms.rst (hunk 5 above)
Comment (niluohe) [reply doc/tutorials/agl-vms.rst]: yes, it's still have in that version

File: doc/tutorials/agl-vms.rst (hunk 6 above)
Comment (niluohe) [reply doc/tutorials/agl-vms.rst]: yes

File: doc/tutorials/agl-vms.rst (hunk 4 above)
Comment (niluohe) [reply doc/tutorials/agl-vms.rst]: Yes, you are right, I will make changes

File: doc/tutorials/agl-vms.rst [hunk 7]
```diff
@@ -24,9 +24,9 @@ meter, the In-Vehicle Infotainment (IVI) system, and the rear seat
 entertainment (RSE).  For the software, there are three VMs running on
 top of ACRN:
 
-* Clear Linux OS runs as the service OS (SOS) to control the cluster meter,
-* an AGL instance runs as a user OS (UOS) controlling the IVI display, and
-* a second AGL UOS controls the RSE display.
+* Clear Linux OS runs as the service OS (Service VM) to control the cluster meter,
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]:
```suggestion
* Clear Linux OS runs as the Service VM to control the instrument cluster,
```

File: doc/tutorials/agl-vms.rst [hunk 8]
```diff
@@ -24,9 +24,9 @@ meter, the In-Vehicle Infotainment (IVI) system, and the rear seat
...
+* an AGL instance runs as a user OS (User VM) controlling the IVI display, and
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]:
```suggestion
* an AGL instance runs as a User VM controlling the In-Vehicle Infotainment (IVI) display, and
```

File: doc/tutorials/agl-vms.rst [hunk 9]
```diff
@@ -24,9 +24,9 @@ meter, the In-Vehicle Infotainment (IVI) system, and the rear seat
...
+* a second AGL User VM controls the RSE display.
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]:
```suggestion
* a second AGL User VM controls the Rear Seat Entertainment (RSE) display.
```

File: doc/tutorials/agl-vms.rst [hunk 10]
```diff
@@ -145,19 +145,24 @@ Service OS
...
 
-      # swupd verify --fix --picky -m 26200
+      # swupd verify --fix --picky -m 31080
 
-#. Use the ``swupd bundle-add`` command and add needed Clear Linux
-   OS bundles::
+#. Use `acrn_quick_setup.sh <https://github.com/projectacrn/acrn-hypervisor/blob/84c2b8819f479c5e6f4641490ff4bf6004f112d1/doc/getting-started/acrn_quick_setup.sh>`_ 
+   to automatically install ACRN::
+      
+      # sh acrn_quick_setup.sh -s 31080 -i
```
Comment (gvancuts) [inline doc/tutorials/agl-vms.rst]:
```suggestion
      $ sudo ./acrn_quick_setup.sh -s 31080 -i 
```

**Pull request merged**